src/utils/onboarding_slides.ts

```diff
@@ -0,0 +1,39 @@
+export default [
```

Comment:
Hey @Aleirbag04 can we use the translate Text here?

Reply:
Yes! It's done =)